src/remodal.js

```diff
@@ -431,6 +422,20 @@
   }
 
   /**
+   * Generates a string separated by dashes and prefixed with NAMESPACE
+   * @private
+   * @param str, str, ...
+   * @returns str
+   */
+  function namespaced() {
```

Comment:
Well Array creation is hardly a costly action and `join` is even faster than string concatenation on Firefox and Opera — http://jsperf.com/array-join-vs-string-connect